### PR TITLE
Fix leveloffset

### PR DESCRIPTION
**Pull request by maximiliankolb**
⚠️  needs a real review to see if my change behaves the way I expect it to.

Cherry-pick into:

* [x] Foreman 3.1

### CODE REVIEW

File: guides/common/assembly_restoring-server-or-smart-proxy-from-a-backup.adoc [hunk 1]
```diff
@@ -2,8 +2,8 @@ include::modules/con_restoring-server-or-smart-proxy-from-a-backup.adoc[]
 
 include::modules/proc_restoring-from-a-full-backup.adoc[leveloffset=+1]
 
-include::modules/proc_restoring-from-incremental-backups.adoc[leveloffset=1]
+include::modules/proc_restoring-from-incremental-backups.adoc[leveloffset=+1]
```
Comment (melcorr) [inline guides/common/assembly_restoring-server-or-smart-proxy-from-a-backup.adoc:5]: lol, well this certainly is now fixed. OMG!

File: guides/common/assembly_restoring-server-or-smart-proxy-from-a-backup.adoc [hunk 2]
```diff
@@ -2,8 +2,8 @@ include::modules/con_restoring-server-or-smart-proxy-from-a-backup.adoc[]
...
 
-include::modules/con_backup-and-restore-smart-proxy-using-a-virtual-machine-snapshot.adoc[leveloffset=1]
+include::modules/con_backup-and-restore-smart-proxy-using-a-virtual-machine-snapshot.adoc[leveloffset=+1]
```
Comment (melcorr) [inline guides/common/assembly_restoring-server-or-smart-proxy-from-a-backup.adoc:7]: lol this one is fixed too!

File: guides/common/assembly_restoring-server-or-smart-proxy-from-a-backup.adoc [hunk 3]
```diff
@@ -2,8 +2,8 @@ include::modules/con_restoring-server-or-smart-proxy-from-a-backup.adoc[]
...
 
-include::modules/proc_synchronizing-an-external-smart-proxy.adoc[leveloffset=2]
+include::modules/proc_synchronizing-an-external-smart-proxy.adoc[leveloffset=+2]
```
Comment (melcorr) [inline guides/common/assembly_restoring-server-or-smart-proxy-from-a-backup.adoc:9]: This one looks out of place? Anything three levels down will be buried. 
10.3.1. Synchronizing an External Smart Proxy

File: guides/doc-Upgrading_and_Updating/master.adoc [hunk 4]
```diff
@@ -70,7 +70,7 @@ include::topics/removing_satellite_tools_repository.adoc[leveloffset=+2]
 endif::[]
 
 // Reclaiming Database Space
-include::common/modules/proc_reclaiming-postgresql-space.adoc[leveloffset=2]
+include::common/modules/proc_reclaiming-postgresql-space.adoc[leveloffset=+2]
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/master.adoc:73]: I see no change here?

File: guides/common/assembly_managing-users-and-roles.adoc [hunk 5]
```diff
@@ -34,7 +34,7 @@ include::modules/proc_creating-a-complete-permission-table.adoc[leveloffset=+2]
 
 include::modules/proc_removing-a-role.adoc[leveloffset=+2]
 
-include::modules/ref_predefined-roles.adoc[leveloffset=2]
+include::modules/ref_predefined-roles.adoc[leveloffset=+2]
```
Comment (melcorr) [inline guides/common/assembly_managing-users-and-roles.adoc:37]: This one is now hidden, lost )5.4.7. Predefined Roles Available in Foreman - what's on nightly branch looks better

**Review by melcorr — CHANGES_REQUESTED** [review]
Interesting... Some yes, some no. 
Did you test locally?

File: guides/common/assembly_restoring-server-or-smart-proxy-from-a-backup.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/common/assembly_restoring-server-or-smart-proxy-from-a-backup.adoc:9]: Yes, but IHMO it looks off on nightly:

```
[10. Restoring Foreman server or Smart Proxy server from a Backup](https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Restoring_Server_or_Smart_Proxy_from_a_Backup_admin)
[10.1. Restoring from a Full Backup](https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Restoring_from_a_Full_Backup_admin)
[11. Restoring from Incremental Backups](https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Restoring_from_Incremental_Backups_admin)
[12. Backup and Restore Smart Proxy server Using a Virtual Machine Snapshot](https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Backup_and_Restore_Smart_Proxy_Using_a_Virtual_Machine_Snapshot_admin)
[12.1. Synchronizing an External Smart Proxy](https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Synchronizing_an_External_Smart_Proxy_admin)
```

vs on my feature branch:

```
[10. Restoring Foreman server or Smart Proxy server from a Backup](https://github.com/theforeman/foreman-documentation/pull/1077#Restoring_Server_or_Smart_Proxy_from_a_Backup_admin)
[10.1. Restoring from a Full Backup](https://github.com/theforeman/foreman-documentation/pull/1077#Restoring_from_a_Full_Backup_admin)
[10.2. Restoring from Incremental Backups](https://github.com/theforeman/foreman-documentation/pull/1077#Restoring_from_Incremental_Backups_admin)
[10.3. Backup and Restore Smart Proxy server Using a Virtual Machine Snapshot](https://github.com/theforeman/foreman-documentation/pull/1077#Backup_and_Restore_Smart_Proxy_Using_a_Virtual_Machine_Snapshot_admin)
```

There are many instances where we have content that is not accessible from the navigation which is absolutely fine in my eyes. Our guides and their navs are already long enough.

I view this change as an improvement.

File: guides/doc-Upgrading_and_Updating/master.adoc (hunk 4 above)
Comment (maximiliankolb) [reply guides/doc-Upgrading_and_Updating/master.adoc:73]: Same.

**Comment by maximiliankolb** [comment]
Rebased to master.

**Pull request merged**